mySpider/mySpider/spiders/log_config_log.txt
2019-09-11 02:12:16,791-log.py[line:128]-INFO: Scrapy 1.3.3 started (bot: mySpider)
2019-09-11 02:12:16,792-log.py[line:131]-INFO: Overridden settings: {'BOT_NAME': 'mySpider', 'LOG_LEVEL': 'WARNING', 'NEWSPIDER_MODULE': 'mySpider.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['mySpider.spiders']}
2019-09-11 02:12:16,852-middleware.py[line:53]-INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2019-09-11 02:12:16,925-middleware.py[line:53]-INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2019-09-11 02:12:16,931-middleware.py[line:53]-INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2019-09-11 02:12:17,035-middleware.py[line:53]-INFO: Enabled item pipelines:
['mySpider.pipelines.MyspiderPipeline', 'mySpider.pipelines.MyspiderPipeline1']
2019-09-11 02:12:17,036-engine.py[line:258]-INFO: Spider opened
2019-09-11 02:12:17,051-logstats.py[line:48]-INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-09-11 02:12:17,053-telnet.py[line:60]-DEBUG: Telnet console listening on 127.0.0.1:6023
2019-09-11 02:12:17,939-engine.py[line:240]-DEBUG: Crawled (200) <GET https://meizitu.com/robots.txt> (referer: None)
2019-09-11 02:12:18,198-engine.py[line:240]-DEBUG: Crawled (200) <GET https://meizitu.com/a/list_1_1.html> (referer: None)
2019-09-11 02:12:19,397-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/robots.txt> (referer: None)
2019-09-11 02:12:19,803-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5521.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:12:19,921-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:20,334-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/01.jpg HTTP/1.1" 200 253847
2019-09-11 02:12:25,980-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:26,363-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/01.jpg HTTP/1.1" 200 253847
2019-09-11 02:12:33,098-scraper.py[line:158]-ERROR: Spider error processing <GET https://www.meizitu.com/a/5521.html> (referer: https://meizitu.com/a/list_1_1.html)
Traceback (most recent call last):
  File "/anaconda3/envs/tensorflow/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 57, in parse_detail
    self.save_img(root,name,src)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 74, in save_img
    print("%s 图片保存完成,用时％2l" % (root_name, end - start))
TypeError: not all arguments converted during string formatting
2019-09-11 02:12:33,132-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5508.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:12:33,240-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:33,654-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/08/01.jpg HTTP/1.1" 200 155450
2019-09-11 02:12:41,608-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:42,067-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/08/01.jpg HTTP/1.1" 200 155450
2019-09-11 02:12:46,255-scraper.py[line:158]-ERROR: Spider error processing <GET https://www.meizitu.com/a/5508.html> (referer: https://meizitu.com/a/list_1_1.html)
Traceback (most recent call last):
  File "/anaconda3/envs/tensorflow/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 57, in parse_detail
    self.save_img(root,name,src)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 74, in save_img
    print("%s 图片保存完成,用时％2l" % (root_name, end - start))
TypeError: not all arguments converted during string formatting
2019-09-11 02:12:46,287-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5509.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:12:46,302-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5505.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,318-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5507.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,332-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5504.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,347-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5503.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,362-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5506.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,377-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5502.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,393-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5501.html> (failed 1 times): [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly.>]
2019-09-11 02:12:46,405-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:46,785-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/09/01.jpg HTTP/1.1" 200 139922
2019-09-11 02:12:50,026-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:50,411-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/09/01.jpg HTTP/1.1" 200 139922
2019-09-11 02:12:53,740-scraper.py[line:158]-ERROR: Spider error processing <GET https://www.meizitu.com/a/5509.html> (referer: https://meizitu.com/a/list_1_1.html)
Traceback (most recent call last):
  File "/anaconda3/envs/tensorflow/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 57, in parse_detail
    self.save_img(root,name,src)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 74, in save_img
    print("%s 图片保存完成,用时％2l" % (root_name, end - start))
TypeError: not all arguments converted during string formatting
2019-09-11 02:12:53,764-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5510.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:12:53,880-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:54,258-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/10/01.jpg HTTP/1.1" 200 158717
2019-09-11 02:12:58,365-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:12:58,745-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/03/10/01.jpg HTTP/1.1" 200 158717
2019-09-11 02:13:02,506-scraper.py[line:158]-ERROR: Spider error processing <GET https://www.meizitu.com/a/5510.html> (referer: https://meizitu.com/a/list_1_1.html)
Traceback (most recent call last):
  File "/anaconda3/envs/tensorflow/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 57, in parse_detail
    self.save_img(root,name,src)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 74, in save_img
    print("%s 图片保存完成,用时％2l" % (root_name, end - start))
TypeError: not all arguments converted during string formatting
2019-09-11 02:13:02,536-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5495.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:13:02,551-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5493.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,564-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5492.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,578-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5512.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,591-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5514.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,604-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5511.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,618-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5513.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:02,650-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:13:03,033-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/02/03/01.jpg HTTP/1.1" 200 101656
2019-09-11 02:13:05,577-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:13:05,968-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/02/03/01.jpg HTTP/1.1" 200 101656
2019-09-11 02:13:07,334-crawler.py[line:247]-INFO: Received SIG_UNBLOCK, shutting down gracefully. Send again to force 
2019-09-11 02:13:07,553-crawler.py[line:254]-INFO: Received SIG_UNBLOCK twice, forcing unclean shutdown
2019-09-11 02:13:08,693-scraper.py[line:158]-ERROR: Spider error processing <GET https://www.meizitu.com/a/5495.html> (referer: https://meizitu.com/a/list_1_1.html)
Traceback (most recent call last):
  File "/anaconda3/envs/tensorflow/lib/python3.6/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks
    current.result = callback(current.result, *args, **kw)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 57, in parse_detail
    self.save_img(root,name,src)
  File "/Users/wangshaohua/Documents/python/scrapy/mySpider/mySpider/spiders/meizitu.py", line 74, in save_img
    print("%s 图片保存完成,用时％2l" % (root_name, end - start))
TypeError: not all arguments converted during string formatting
2019-09-11 02:13:08,718-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5496.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:13:08,724-engine.py[line:297]-INFO: Closing spider (shutdown)
2019-09-11 02:13:08,738-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5515.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:08,752-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5498.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:08,769-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5499.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:08,782-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5500.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:08,795-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5494.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl3_read_bytes', 'ssl handshake failure')]>]
2019-09-11 02:13:08,808-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5516.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:08,822-retry.py[line:68]-DEBUG: Retrying <GET https://www.meizitu.com/a/5497.html> (failed 1 times): [<twisted.python.failure.Failure OpenSSL.SSL.Error: [('SSL routines', 'ssl23_read', 'ssl handshake failure')]>]
2019-09-11 02:13:41,132-log.py[line:128]-INFO: Scrapy 1.3.3 started (bot: mySpider)
2019-09-11 02:13:41,133-log.py[line:131]-INFO: Overridden settings: {'BOT_NAME': 'mySpider', 'LOG_LEVEL': 'WARNING', 'NEWSPIDER_MODULE': 'mySpider.spiders', 'ROBOTSTXT_OBEY': True, 'SPIDER_MODULES': ['mySpider.spiders']}
2019-09-11 02:13:41,202-middleware.py[line:53]-INFO: Enabled extensions:
['scrapy.extensions.corestats.CoreStats',
 'scrapy.extensions.telnet.TelnetConsole',
 'scrapy.extensions.logstats.LogStats']
2019-09-11 02:13:41,271-middleware.py[line:53]-INFO: Enabled downloader middlewares:
['scrapy.downloadermiddlewares.robotstxt.RobotsTxtMiddleware',
 'scrapy.downloadermiddlewares.httpauth.HttpAuthMiddleware',
 'scrapy.downloadermiddlewares.downloadtimeout.DownloadTimeoutMiddleware',
 'scrapy.downloadermiddlewares.defaultheaders.DefaultHeadersMiddleware',
 'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware',
 'scrapy.downloadermiddlewares.retry.RetryMiddleware',
 'scrapy.downloadermiddlewares.redirect.MetaRefreshMiddleware',
 'scrapy.downloadermiddlewares.httpcompression.HttpCompressionMiddleware',
 'scrapy.downloadermiddlewares.redirect.RedirectMiddleware',
 'scrapy.downloadermiddlewares.cookies.CookiesMiddleware',
 'scrapy.downloadermiddlewares.stats.DownloaderStats']
2019-09-11 02:13:41,277-middleware.py[line:53]-INFO: Enabled spider middlewares:
['scrapy.spidermiddlewares.httperror.HttpErrorMiddleware',
 'scrapy.spidermiddlewares.offsite.OffsiteMiddleware',
 'scrapy.spidermiddlewares.referer.RefererMiddleware',
 'scrapy.spidermiddlewares.urllength.UrlLengthMiddleware',
 'scrapy.spidermiddlewares.depth.DepthMiddleware']
2019-09-11 02:13:41,380-middleware.py[line:53]-INFO: Enabled item pipelines:
['mySpider.pipelines.MyspiderPipeline', 'mySpider.pipelines.MyspiderPipeline1']
2019-09-11 02:13:41,380-engine.py[line:258]-INFO: Spider opened
2019-09-11 02:13:41,396-logstats.py[line:48]-INFO: Crawled 0 pages (at 0 pages/min), scraped 0 items (at 0 items/min)
2019-09-11 02:13:41,398-telnet.py[line:60]-DEBUG: Telnet console listening on 127.0.0.1:6023
2019-09-11 02:13:42,288-engine.py[line:240]-DEBUG: Crawled (200) <GET https://meizitu.com/robots.txt> (referer: None)
2019-09-11 02:13:42,550-engine.py[line:240]-DEBUG: Crawled (200) <GET https://meizitu.com/a/list_1_1.html> (referer: None)
2019-09-11 02:13:43,543-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/robots.txt> (referer: None)
2019-09-11 02:13:43,915-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5521.html> (referer: https://meizitu.com/a/list_1_1.html)
2019-09-11 02:13:44,031-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:13:44,540-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/01.jpg HTTP/1.1" 200 253847
2019-09-11 02:14:31,523-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:14:31,904-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/02.jpg HTTP/1.1" 200 144368
2019-09-11 02:14:36,795-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:14:37,179-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/03.jpg HTTP/1.1" 200 162479
2019-09-11 02:14:40,810-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:14:41,190-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/04.jpg HTTP/1.1" 200 358303
2019-09-11 02:14:48,624-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:14:49,154-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/05.jpg HTTP/1.1" 200 96365
2019-09-11 02:14:56,994-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:14:57,443-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/06.jpg HTTP/1.1" 200 132234
2019-09-11 02:15:01,057-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:15:01,470-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/07.jpg HTTP/1.1" 200 106030
2019-09-11 02:15:10,481-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:15:10,859-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/08.jpg HTTP/1.1" 200 172871
2019-09-11 02:15:14,805-connectionpool.py[line:205]-DEBUG: Starting new HTTP connection (1): pic.topmeizi.com:80
2019-09-11 02:15:15,188-connectionpool.py[line:393]-DEBUG: http://pic.topmeizi.com:80 "GET /wp-content/uploads/2017a/04/08/09.jpg HTTP/1.1" 200 98375
2019-09-11 02:15:17,802-engine.py[line:240]-DEBUG: Crawled (200) <GET https://www.meizitu.com/a/5512.html> (referer: https://meizitu.com/a/list_1_1.html)

myspider/settings.py
# -*- coding: utf-8 -*-

# Scrapy settings for myspider project
#
# For simplicity, this file contains only settings considered important or
# commonly used. You can find more settings consulting the documentation:
#
#     http://doc.scrapy.org/en/latest/topics/settings.html
#     http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
#     http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html

BOT_NAME = 'myspider'

SPIDER_MODULES = ['myspider.spiders']
NEWSPIDER_MODULE = 'myspider.spiders'

LOG_LEVEL = 'INFO'
LOG_FILE = './log.log'


# Crawl responsibly by identifying yourself (and your website) on the user-agent
USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36'

# Obey robots.txt rules
ROBOTSTXT_OBEY = True

# Configure maximum concurrent requests performed by Scrapy (default: 16)
#CONCURRENT_REQUESTS = 32

# Configure a delay for requests for the same website (default: 0)
# See http://scrapy.readthedocs.org/en/latest/topics/settings.html#download-delay
# See also autothrottle settings and docs
DOWNLOAD_DELAY = 3
# The download delay setting will honor only one of:
CONCURRENT_REQUESTS_PER_DOMAIN = 2
CONCURRENT_REQUESTS_PER_IP = 2

# Disable cookies (enabled by default)
#COOKIES_ENABLED = False

# Disable Telnet Console (enabled by default)
#TELNETCONSOLE_ENABLED = False

# Override the default request headers:
#DEFAULT_REQUEST_HEADERS = {
#   'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',
#   'Accept-Language': 'en',
#}

# Enable or disable spider middlewares
# See http://scrapy.readthedocs.org/en/latest/topics/spider-middleware.html
#SPIDER_MIDDLEWARES = {
#    'myspider.middlewares.MyspiderSpiderMiddleware': 543,
#}

# Enable or disable downloader middlewares
# See http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
DOWNLOADER_MIDDLEWARES = {
   'myspider.middlewares.RandomUserAgentMiddleware': 543,
   'myspider.middlewares.CheckUserAgent': 544,
   'myspider.middlewares.AoisolasSpiderMiddleware': 1,

}

# Enable or disable extensions
# See http://scrapy.readthedocs.org/en/latest/topics/extensions.html
#EXTENSIONS = {
#    'scrapy.extensions.telnet.TelnetConsole': None,
#}

# Configure item pipelines
# See http://scrapy.readthedocs.org/en/latest/topics/item-pipeline.html
# 设置存放图片的路径
# IMAGES_STORE = '/root/img'
IMAGES_STORE = '/Users/wangshaohua/Desktop/dev/myspider/upload'


# 90天的图片失效期限
IMAGES_EXPIRES = 90

# 设置生成缩略图。
# IMAGES_THUMBS = {
#     'small': (50, 50),   # (宽， 高)
#     'big': (270, 270),
# }

ITEM_PIPELINES = {
   'myspider.pipelines.MyspiderPipeline': 300,
   'myspider.pipelines.SaveSqlPipeline': 301,
}

# Enable and configure the AutoThrottle extension (disabled by default)
# See http://doc.scrapy.org/en/latest/topics/autothrottle.html
AUTOTHROTTLE_ENABLED = True
# The initial download delay
AUTOTHROTTLE_START_DELAY = 5
# The maximum download delay to be set in case of high latencies
AUTOTHROTTLE_MAX_DELAY = 60
# The average number of requests Scrapy should be sending in parallel to
# each remote server
AUTOTHROTTLE_TARGET_CONCURRENCY = 1.0
# Enable showing throttling stats for every response received:
AUTOTHROTTLE_DEBUG = False


# Enable and configure HTTP caching (disabled by default)
# See http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html#httpcache-middleware-settings
#HTTPCACHE_ENABLED = True
#HTTPCACHE_EXPIRATION_SECS = 0
#HTTPCACHE_DIR = 'httpcache'
#HTTPCACHE_IGNORE_HTTP_CODES = []
#HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'

USER_AGENT_LIST=[
    "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.1 (KHTML, like Gecko) Chrome/22.0.1207.1 Safari/537.1",
    "Mozilla/5.0 (X11; CrOS i686 2268.111.0) AppleWebKit/536.11 (KHTML, like Gecko) Chrome/20.0.1132.57 Safari/536.11",
    "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/536.6 (KHTML, like Gecko) Chrome/20.0.1092.0 Safari/536.6",
    "Mozilla/5.0 (Windows NT 6.2) AppleWebKit/536.6 (KHTML, like Gecko) Chrome/20.0.1090.0 Safari/536.6",
    "Mozilla/5.0 (Windows NT 6.2; WOW64) AppleWebKit/537.1 (KHTML, like Gecko) Chrome/19.77.34.5 Safari/537.1",
    "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/536.5 (KHTML, like Gecko) Chrome/19.0.1084.9 Safari/536.5",
    "Mozilla/5.0 (Windows NT 6.0) AppleWebKit/536.5 (KHTML, like Gecko) Chrome/19.0.1084.36 Safari/536.5",
    "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1063.0 Safari/536.3",
    "Mozilla/5.0 (Windows NT 5.1) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1063.0 Safari/536.3",
    "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1; Trident/4.0; SE 2.X MetaSr 1.0; SE 2.X MetaSr 1.0; .NET CLR 2.0.50727; SE 2.X MetaSr 1.0)",
    "Mozilla/5.0 (Windows NT 6.2) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1062.0 Safari/536.3",
    "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1062.0 Safari/536.3",
    "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1; 360SE)",
    "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1061.1 Safari/536.3",
    "Mozilla/5.0 (Windows NT 6.1) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1061.1 Safari/536.3",
    "Mozilla/5.0 (Windows NT 6.2) AppleWebKit/536.3 (KHTML, like Gecko) Chrome/19.0.1061.0 Safari/536.3",
    "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/535.24 (KHTML, like Gecko) Chrome/19.0.1055.1 Safari/535.24",
    "Mozilla/5.0 (Windows NT 6.2; WOW64) AppleWebKit/535.24 (KHTML, like Gecko) Chrome/19.0.1055.1 Safari/535.24",
    "Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Win64; x64; Trident/5.0; .NET CLR 3.5.30729; .NET CLR 3.0.30729; .NET CLR 2.0.50727; Media Center PC 6.0)",
    "Mozilla/5.0 (compatible; MSIE 8.0; Windows NT 6.0; Trident/4.0; WOW64; Trident/4.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; .NET CLR 1.0.3705; .NET CLR 1.1.4322)",
    "Mozilla/4.0 (compatible; MSIE 7.0b; Windows NT 5.2; .NET CLR 1.1.4322; .NET CLR 2.0.50727; InfoPath.2; .NET CLR 3.0.04506.30)",
    "Mozilla/5.0 (Windows; U; Windows NT 5.1; zh-CN) AppleWebKit/523.15 (KHTML, like Gecko, Safari/419.3) Arora/0.3 (Change: 287 c9dfb30)",
    "Mozilla/5.0 (X11; U; Linux; en-US) AppleWebKit/527+ (KHTML, like Gecko, Safari/419.3) Arora/0.6",
    "Mozilla/5.0 (Windows; U; Windows NT 5.1; en-US; rv:1.8.1.2pre) Gecko/20070215 K-Ninja/2.1.1",
    "Mozilla/5.0 (Windows; U; Windows NT 5.1; zh-CN; rv:1.9) Gecko/20080705 Firefox/3.0 Kapiko/3.0",
    "Mozilla/5.0 (X11; Linux i686; U;) Gecko/20070322 Kazehakase/0.4.5"
]

# mysql 数据库配置
MYSQL_HOST = 'localhost'
MYSQL_PORT = 3306
MYSQL_DATABASE = 'meizitu'
MYSQL_USER = 'root'
MYSQL_PASSWORD = '123456'
MYSQL_CHARSET = 'utf8'

FDSF_CONF = 'myspider/util/client.conf'
FDSF_NGINX = 'http://39.96.63.98:8888/'